File: 백준/Bronze/10872. 팩토리얼/팩토리얼.py
import sys

input = sys.stdin.readline
write = sys.stdout.write

def solve():
   line = input().rstrip()
   if not line: return
   n = int(line)
   
   result = 1
   for i in range(1, n + 1):
       result *= i
   
   write(str(result) + "\n")
        
if __name__ == "__main__":
    solve()

File: 백준/Bronze/10872. 팩토리얼/팩토리얼.cc
#include <stdio.h>

int main(void) {
	int n = 0;
	if (scanf("%d", &n) != 1) return 0;
	if (0 > n || n > 12) return 0;

	int result = 1;
	for (int i = 1; i <= n; i++) {
		result *= i;
	}

	printf("%d\n", result);

	return 0;
}